pkgs/development/embedded/blisp/default.nix

```diff
@@ -31,12 +32,16 @@ stdenv.mkDerivation {
     "-DBLISP_USE_SYSTEM_LIBRARIES=ON"
   ];
 
+  passthru.tests.version = testers.testVersion {
+    package = finalAttrs.finalPackage;
+    command = "blisp --version";
```

Comment:
If I am not mistaken, this is already the default ( with the `mainProgram` change below). can you check if it still work if you remove this line?

Reply:
Yes. Indeed it does default to: `"${package.meta.mainProgram or package.pname or package.name} --version"`.

TIL.